sections/queueing_model/waiting_time/main.tex

```diff
@@ -62,21 +62,31 @@ \subsection{Waiting time} \label{sec:waiting_time}
 simulated waiting time using discrete event simulation over a range of values of 
 \(\lambda_2\) (details of the discrete event simulation model are given in 
 appendix~\ref{sec:appendix_des}).
+The figure is used to demonstrate the accuracy of the waiting time formula of
+the constructed queueing model as well as the effect of truncating the model.
 The simulation was ran 100 times and the recorded mean waiting time at each 
 iteration is used to populate the violin plots.
-The waiting times generated by the simulation match the ones generated by the 
-Markov chains model.
+In detail, figure \ref{fig:markov_vs_des_waiting_time_comparison_overall} shows
+the calculated mean waiting time using Markov chain, using a truncated 
+simulation and using a simulation with infinite capacity (without the artificial
+parameters \(N\) and \(M\)).
+Each plot corresponds to different values of \(N\) and \(M\) and is ran over 
+different values of \(\lambda_2\).
+The untruncated simulation values (blue) are the same at all three graphs since
```

Comment:
```suggestion
The untruncated simulation values are the same at all three graphs since
```

Ideally we want to avoid colors all together (inclusivity and the printed version will probably be in black and white). This might  mean we need tog greyscale the plots.

Reply:
Noted